### DISCUSSION TITLE
Encoders without detents CountMode

### DISCUSSION THREAD

**Discussion by Electrotechnique** [luni64/EncoderTool#12] [Q&A]
Hello, I'm using four encoders without detents - Bourns [PEC11R-4020F-S0024-ND](https://www.digikey.com/en/products/detail/bourns-inc/PEC11R-4020F-S0024/4499667) multiplexed with three 74HC165s. Are you supposed to use CountMode quarter with these? Full mode works but is unstable and sometimes decrements when it's supposed to be incrementing. Thanks.

**Comment by luni64** [comment]
Full CountMode can only guarantee that the counts are correct after the encoder has been rotated. The bounces during the rotation will lead to rapid up/down counts which you probably saw. But, as mentioned the end position should be correct. 

For an encoder without detents you can also choose Half and Quarter count modes.  Both modes ensure that you will not get rapid up/down counts while the switches bounce. The price for that is the reduced resolution of course.

**Reply by Electrotechnique** [reply]
Thanks very much.
